openedx/features/caliper_tracking/transformers/navigation_transformers.py

```diff
@@ -0,0 +1,20 @@
+import json
+
+def edx_ui_lms_link_clicked(current_event, caliper_event):
+	caliper_event["type"] = "NavigationEvent"
+	caliper_event["action"] = "NavigateTo"
+	caliper_event["actor"]["type"] = "Person"
+
+	current_event_details = json.loads(current_event["event"])
+
+	caliper_event["object"] = {
+	    "id": current_event_details["target_url"],
+	    "type": "WebPage",
+	}
+
+	caliper_event["referrer"] = {
```

Comment:
Please indent the code like you've done above.
```suggestion
	caliper_event["referrer"] = {
	    "id": current_event["referrer"],
	    "type": "WebPage",
	}
```